src/main/java/com/ldbc/driver/workloads/ldbc/snb/bi/SerializationUtil.java

```diff
@@ -12,17 +12,35 @@
 public class SerializationUtil
 {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-    private static final TypeReference<List<List<Object>>> LIST_OF_LISTS_TYPE_REFERENCE =
+    private static final TypeReference LIST_OF_LISTS_TYPE_REFERENCE =
             new TypeReference<List<List<Object>>>()
             {
             };
+    private static final TypeReference<List<List<List<Long>>>> LIST_OF_LISTS_TYPE_REFERENCE_FOR_LONGS =
+            new TypeReference<List<List<List<Long>>>>()
+            {
+            };
+
 
     public static synchronized List<List<Object>> marshalListOfLists( String serializedJson )
             throws SerializingMarshallingException
     {
+        return marshalListOfLists( serializedJson, LIST_OF_LISTS_TYPE_REFERENCE );
+    }
+
+    public static synchronized List<List<Object>> marshalListOfListsLongs( String serializedJson )
```

Comment:
do we really need this? the method returns `List<List<Object>>` anyway, and the consumer of this method casts. is this not unnecessary?

Reply:
Well technically, the consumer cannot cast, as it will result in a ClassCastException. So the consumer would need to iterate through the list and for each element `e` invoke `Long.valueOf(e)`.

Reply:
See also #72

Reply:
ok 👍